src/main/kotlin/apply/ui/admin/mission/MissionForm.kt

```diff
@@ -20,26 +23,51 @@ class MissionForm() : BindingIdentityFormLayout<MissionData>(MissionData::class)
     }
     private val startDateTime: DateTimePicker = DateTimePicker("시작 일시")
     private val endDateTime: DateTimePicker = DateTimePicker("종료 일시")
+
+    private var judgeItemId: Long = 0L
+    private val testName: TextField? = TextField("채점에 사용할 미션 이름")
+    private val programmingLanguage: Select<ProgrammingLanguage>? =
+        createItemSelect<ProgrammingLanguage>("언어 선택").apply {
+            setItems(*ProgrammingLanguage.values())
+            setItemLabelGenerator { it.language }
+        }
+    private val evaluationItem: Select<EvaluationItemData>? = createItemSelect<EvaluationItemData>("평가 항목").apply {
+        setItemLabelGenerator(EvaluationItemData::title)
+        isEmptySelectionAllowed = false
+    }
+
     private val submittable: RadioButtonGroup<Boolean> = createBooleanRadioButtonGroup("제출 여부", "제출 시작", "제출 중지", false)
     private val hidden: RadioButtonGroup<Boolean> = createBooleanRadioButtonGroup("공개 여부", "비공개", "공개", true)
 
     init {
-        add(title, evaluation, startDateTime, endDateTime, description, submittable, hidden)
+        add(
+            title, evaluation, startDateTime, endDateTime, description,
+            Hr(), testName, programmingLanguage, evaluationItem, Hr(),
+            submittable, hidden
+        )
         setResponsiveSteps(ResponsiveStep("0", 1))
         drawRequired()
     }
 
     constructor(
-        evaluations: List<EvaluationSelectData>
+        evaluations: List<EvaluationSelectData>,
+        listener: (id: Long) -> List<EvaluationItemData>,
     ) : this() {
         evaluation.setItems(evaluations)
+        evaluation.addValueChangeListener {
+            val evaluationItems: List<EvaluationItemData> = mutableListOf()
+            evaluationItem?.setItems(evaluationItems.plus(listener(it.value.id)))
```

Comment:
```suggestion
            evaluationItem?.setItems(evaluationItems +listener(it.value.id))
```

<img width="659" alt="image" src="https://user-images.githubusercontent.com/52564093/190866266-3c40214c-9d6a-40e4-8da0-92330ca33b9e.png">

r: plus는 operator function입니다~